crates/wasm/build.sh

```diff
@@ -0,0 +1,3 @@
+RUSTFLAGS='-C target-feature=+atomics,+bulk-memory,+mutable-globals' \
+    rustup run nightly \
+    wasm-pack build --target web . -- -Zbuild-std=panic_abort,std
```

Comment:
The command in https://github.com/tlsnotary/tlsn/blob/dev/crates/wasm/README.md doesn't have `--`, should we stick to one?